frontend/src/components/admin/user-management/admin/AddAdminModal/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
can we combine this file with `UsersPageHeader`?

Reply:
We could, but I feel like it's nice to have a logical grouping of the two modals